src/usr/local/www/interfaces_ppps_edit.php

```diff
@@ -671,6 +680,13 @@ function build_country_list() {
 					'Check the "Configure NULL" box to configure a blank Service name.');
 
 	$section->add($group);
+
+	$section->addInput(new Form_Input(
+		'pppoe_host-uniq',
+		'Host-Uniq',
+		'text',
+		$pconfig['pppoe_host-uniq']
+	))->setHelp('This field can usually be left empty.');
```

Comment:
I'd like to see a description of what this option does rather than just "Can usually be left blank"